mpu9250/transport.go

```diff
@@ -0,0 +1,148 @@
+// Copyright 2020 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package mpu9250
+
+import (
+	"fmt"
+
+	"periph.io/x/conn/v3/gpio"
+	"periph.io/x/conn/v3/i2c"
+	"periph.io/x/conn/v3/physic"
+	"periph.io/x/conn/v3/spi"
+	"periph.io/x/conn/v3/spi/spireg"
+)
+
+// DebugF the debug function type.
+type DebugF func(string, ...interface{})
```

Comment:
Side note: we should probably standardize on the log/slog package. It's actually pretty good.